configs/test-utils/src/ThrowError.tsx

```diff
@@ -1,11 +1,21 @@
 import type { PropsWithChildren } from 'react'
-import { useEffect, useState } from 'react'
+import { useEffect, useLayoutEffect, useRef, useState } from 'react'
+
+const isClient = typeof window !== 'undefined'
+export const useIsomorphicLayoutEffect = isClient ? useLayoutEffect : useEffect
+
+export const useTimeout = (fn: () => void, ms: number) => {
```

Comment:
I renamed useSetTimeout to useTimeout, because we use just Timeout not setTimeout

Reply:
How about we use ms package for this? https://www.npmjs.com/package/ms
It is really good for dx.

Reply:
I understood that you meant to use `ms` inside `suspensive`. But wouldn't it be more useful to use it like the code below? If `ms` is unnecessary, it can prevent unnecessary dependencies from being installed and unnecessary calculations to be performed. Also, the unit of time desired by the Delay component is quite small. In most cases, I expect to use `ms prop` for Delay of less than 2000ms, but I am wondering whether it would be a good idea to use `ms` internally in this situation. If you need to include it, I think it would be a good idea to create `ms` yourself so that you can include only the necessary parts.

### Delay using with ms is okay
```tsx
import ms from 'ms'

<Delay ms={ms('2 days')}>
  {/* ... */}
</Delay>
```

### Delay have to make BREAKING CHANGE if we use `ms` internally
```tsx
import ms from 'ms'

<Delay time={'2 days'}> // this ms prop have to make BREAKING CHANGE. because '2 days' is not just ms prop
  {/* ... */}
</Delay>
```

Reply:
Oh that makes sense. I agree `ms` can be good devDependency to test case like this!